app/controllers/content/vic_local_uploads_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Content
+  class VICLocalUploadsController < ApplicationController
+    skip_before_action :authenticate
+
+    # This controller is used for simulating the reverse proxy
+    # photo upload retreival for VIC development. Do not use this
+    # in production code
+    # :nocov:
+    def find_file
+      if Rails.env.development?
```

Comment:
@jkassemi I added a sanitization method and also wrapped the code to ensure we never serve a file unless we're in development. Let me know if you think we need more around this.

Reply:
Cool - three additional mitigations, looks good!